tao_compiler/mlir/disc/transforms/input_inline_fusion_pattern.cc

```diff
@@ -175,7 +175,7 @@ LogicalResult InputInlineFusionPattern::processParallelOp(
     // Clean all the ops that do not have LoadOps inside the nested
     // ParallelOps and is not the ancestor of any ops that have LoadOps
     // inside the nested ParallelOps.
-    cleanUnusedLhloOps(parent_block);
+    cleanUnusedLhloOps(parent_block, &rewriter);
```

Comment:
Are there other usages of cleanUnusedLhloOps elsewhere?

Reply:
![image](https://user-images.githubusercontent.com/8392447/180908761-c6442353-dd37-4cb7-bd54-f20153b643cc.png)

There is another cleanUnusedLhloOps definition will it introduce problems too?

Reply:
> Are there other usages of cleanUnusedLhloOps elsewhere?

These are not inside a rewrite pattern.

Reply:
> ![image](https://user-images.githubusercontent.com/8392447/180908761-c6442353-dd37-4cb7-bd54-f20153b643cc.png)
> 
> There is another cleanUnusedLhloOps definition will it introduce problems too?

seems that it is a duplicate implementation. I'll remove it in the next PR.

Reply:
I double-check the code: it's inside tf_community, not inside disc. I think it's the code we upstream to the community in the last year.